_data/projects/POSMS.yml

```diff
@@ -0,0 +1,9 @@
+name: POSMS
+desc: A free, open source subscriber system
+site: 
```

Comment:
Build is failing because you didn't specify `site`. If you don't have a site, just repeat the GitHub link here.